### DISCUSSION TITLE
Basic questions

### DISCUSSION THREAD

**Discussion by Adiventure** [tiredofit/docker-freepbx#190] [Q&A]
As a background, I'm totally new to PBXs, but have mused over the idea of setting one up. I'm now a few days into having a running unraid server and remembered I have a Asterisk card I bought a few years ago to play with (and then immediately hit a roadblock with it not being supported in Windows). Can one connect a FXO/FXS card and use it with this container?

**Comment by xWeegix** [comment]
I also would like to know this and am not too experienced with docker hardware access. I'm not sure what the issues would be outside of maybe unraid needing some way to handle interacting with the card. It looks like DAHDI is what provides the drivers but I'm not sure how that would need to be implemented to work with the freepbx docker. There is a github docker project at /ikle/dahdi-linux that talks about compiling a kernel module to enable support.
